src/backend/SyncBackendConnection.cpp

```diff
@@ -1156,4 +1156,74 @@ std::vector<std::string> SyncBackendConnection::ds_supported_transports()
     return supported_transports;
 }
 
+std::vector<std::string> SyncBackendConnection::get_statistic_kinds()
+{
+#ifdef DEBUG
+    std::vector<std::string> statistic_kinds = {
+        "NONE",
+        "MEAN",
+        "STANDARD_DEVIATION",
+        "MAX",
+        "MIN",
+        "MEDIAN",
+        "COUNT",
+        "SUM"
+    };
+#else
+    std::vector<std::string> statistic_kinds = {
+        "NONE",
+        "MEAN",
+        "STANDARD_DEVIATION",
+        "MAX",
+        "MIN",
+        "MEDIAN",
+        "SUM"
+    };
+#endif // ifdef DEBUG
+
+    return statistic_kinds;
```

Comment:
In general: Try to not instantiate variables in methods that just return a value.

Check this possibility, or find another workaround for this cases:
```suggestion
    return std::vector<std::string> ({
        "NONE",
        "MEAN",
        "STANDARD_DEVIATION",
        "MAX",
        "MIN",
        "MEDIAN",
#ifdef DEBUG
        "COUNT",
#endif // ifdef DEBUG
        "SUM"
    });
```